<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicVCamConfig</class>
 <widget class="QDialog" name="OBSBasicVCamConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic.VCam.VirtualCamera</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="outputTypeLabel">
     <property name="text">
      <string>Basic.VCam.OutputType</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="outputType"/>
   </item>
   <item>
    <widget class="QLabel" name="outputSelectionLabel">
     <property name="text">
      <string>Basic.VCam.OutputSelection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="outputSelection"/>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Basic.VCam.RestartWarning</string>
     </property>
     <property name="class" stdset="0">
      <string notr="true">text-warning</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OBSBasicVCamConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OBSBasicVCamConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
